Test/WebTestCase.php

```diff
@@ -554,7 +555,7 @@ protected function getUrl($route, $params = array(), $absolute = false)
      *
      * @return void
      */
-    public function isSuccessful($response, $success = true, $type = 'text/html')
+    public function isSuccessful(Response $response, $success = true, $type = 'text/html')
```

Comment:
since others might have extended this method it would be a BC break. then again most users likely didn't extend this method. still imho as we are looking towards a 2.0 release, this change should be in a separate PR that we will merge once we begin on 2.0